crates/holochain_p2p/src/types/actor.rs

```diff
@@ -60,9 +45,6 @@ impl Default for GetOptions {
             timeout_ms: None,
             as_race: true,
```

Comment:
I don't think these two are implemented either but they are something we might want, to send queries for missing data in parallel.

Reply:
None of these are implemented, no. It's a 4 year old idea. I think it's fine to add optimizations as these later, but while they're not exposed, it'd be a breaking change in the serialization anyway, so fine to remove from my POV.